### DISCUSSION TITLE
Images that are hosted elsewhere do not load on 7.0.3

### DISCUSSION THREAD

**Discussion by Wandtket** [unoplatform/Uno.Wasm.Bootstrap#596] [Q&A]
Just load any image with a URL that isn't hosted inside your own application.

`<Image x:Name="F3Before" Source="https://i.imgur.com/KQheCkB.jpg" Stretch="UniformToFill"/>`

**Comment by jeromelaban** [comment]
This is likely to be a CORS or security restriction issue. Does it work when loading the site with HTTPS ?

**Reply by jeromelaban** [reply]
This is in fact related to the new threading support and the "Secure mode" required to get it running. It will be enabled selectively after https://github.com/unoplatform/Uno.Wasm.Bootstrap/pull/601.
